net/remote/remote-node.sh

```diff
@@ -298,6 +299,11 @@ cat >> ~/solana/on-reboot <<EOF
       ./multinode-demo/faucet.sh > faucet.log 2>&1 &
 EOF
     fi
+
+    if [[ "$maybeWenRestart" != "" ]]; then
```

Comment:
`agave-validator help` suggests this is a `DIR` so can use:
```[[ -d $maybeWenRestart ]]```
Or if you don't want to perform validation can use:
```[[ -n $maybeWenRestart ]]```

Reply:
Err, it should be a file not a directory any more. I'll change that. Using -n because it's okay if the file doesn't exist.